nibabel/externals/netcdf.py

```diff
@@ -210,6 +210,13 @@ def __init__(self, filename, mode='r', mmap=None, version=1):
             self.fp = open(self.filename, '%sb' % mode)
             if mmap is None:
                 mmap = True
+        try:
+            self.fp.seek(0, 2)
```

Comment:
I guess the ValueError comes here?  Best to isolate that line with `try`, as in:

```
try:
    self.fp.seek(0, 2)
except ValueError:
    self.file_bytes = -1  # Unknown file length (gzip).
else:
    self.file_bytes = self.fp.tell()
    self.fp.seek(0)
```